cetmix_tower_server/models/constants.py

```diff
@@ -32,3 +32,15 @@
 
 # Returned when the command failed to execute due to a python code execution error
 PYTHON_COMMAND_ERROR = -24
+
+# Returned when the server is not found
+SERVER_NOT_FOUND = -1
```

Comment:
Question: where did you take these codes from?

Reply:
I will change it, to 404